backend/in_memory_storage/in_memory_storage.go

```diff
@@ -31,10 +31,46 @@ type inMemStore struct {
 	settings models.Settings
 }
 
-func MakeInMemStore() storage.StorageService {
+func MakeInMemStore(defaultDomain string) storage.StorageService {
+	defaultPermissionList := models.Permission{Allow: []models.Principal{{
+		Type: models.PrincipalTypeDomain,
+		ID:   defaultDomain,
+	}}}
+	teamPermissions := models.TeamPermissions{
+		Read:  defaultPermissionList,
+		Write: defaultPermissionList,
+	}
+	userAEmail := "userA@userA.com"
```

Comment:
My understanding is that these users are primarily for unit tests, while the passed-in `defaultDomain` is intended for people testing authentication manually when they run the backend in development using `--inmemstore` - is that right?

If so then I think we'd benefit from some comments to that effect.

Reply:
I've added a comment about the users, let me know if that is what you were thinking.

Reply:
Yes, this is better, thanks - though see comments elsewhere.